### ISSUE TITLE
[CoE Starter Kit - QUESTION] Upgrade COE start kit

### ISSUE THREAD

**Issue by bocESMI** [microsoft/coe-starter-kit#7521]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

We are planning to install/re-install the latest COE starter kit in our environment and we would like your advise. Our current COE starter kit was installed 3 years ago version 1.25 - 1.71 depending on the solution package. We have done some customization but not very extensive. I have installed the Jan version of COE (Version 3.XX)on our test tenant and seen many components/functionality that differ quite a bit on what we have installed in PROD now.
Question : Should upgrade the current version or do a fresh install of the COE Starter Kit?  

I am following the Update documentation provided and documenting all customizations and such https://learn.microsoft.com/en-us/power-platform/guidance/coe/after-setup#installing-updates.

### What solution are you experiencing the issue with?

None

### What solution version are you using?

1.7x

### What app or flow are you having the issue with?

CoE Solution

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Yes, unfortunately I think you'll be best served starting from scratch with that age. 

Please note that we now have a Setup and Upgrade Wizard. Its highly recommended that you use it to walk you through setup and upgrades. It will ensure all the new flows are turned on in the correct order, trigger things like population of the bookmarks screen and do other cleanup steps needed.
You should do this both for upgrades and clean installs.
To use it, first perform the import of the solution install (or upgrade) manually, then boot the app and walk through the screens.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b84cc1ff-383a-497c-9236-569e8313c73c)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/ed02f0d0-c276-48cc-90af-47e5bf5a928e)
